spec/line/bot/client_channel_token_spec.rb

```diff
@@ -112,4 +112,15 @@ def generate_client
 
     expect(response).to be_a(Net::HTTPOK)
   end
+
+  it 'verifies ID token' do
+    uri_template = Addressable::Template.new Line::Bot::API::DEFAULT_OAUTH_ENDPOINT + '/oauth2/v2.1/verify'
+    stub_request(:post, uri_template).to_return { |request| {body: '', status: 200} }
```

Comment:
In the test, the body of the response is empty, but it actually returns the contents of the ID token. I think it would be better to add content validation for some values, referring to the "Example response" in the documentation.
https://developers.line.biz/en/reference/line-login/#verify-id-token-response